aten/src/ATen/DLConvertor.cpp

```diff
@@ -129,7 +130,7 @@ static Device getATenDevice(const DLDevice& ctx, void* data) {
       return at::Device(DeviceType::HIP, ctx.device_id);
 #endif
     case DLDeviceType::kDLOneAPI:
-      return at::detail::getXPUHooks().getATenDeviceFromDLPackDevice(ctx, data);
+      return at::detail::getXPUHooks().getDeviceFromPtr(data);
```

Comment:
May I know why not `at::Device(DeviceType::XPU, ctx.device_id)`?

Reply:
at::Device is constructed by the device index of framework presenting the order index of GPU device. But `ctx.device_id` is a global order index containing GPU device or other device.